packages/0x.js/package.json

```diff
@@ -84,7 +84,7 @@
         "@0xproject/utils": "^1.0.11",
         "@0xproject/web3-wrapper": "^3.0.1",
         "ethereum-types": "^1.0.8",
-        "ethers": "3.0.22",
+        "ethers": "^4.0.0-beta.14",
```

Comment:
I'm sure Ethers will introduce breaking changes during their beta that will get caught by `^`. Can we leave it out until they are out of beta?